tests/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+---
+version: '3.2'
+services:
+  azurite:
+    image: mcr.microsoft.com/azure-storage/azurite:3.3.0-preview
+    ports:
+      - "10000"
+  oss:
+    image: rkuprieiev/oss-emulator
```

Comment:
Using my personal repo while waiting to be granted an access to iterativeai dockerhub org :slightly_smiling_face: